Knittery/Network/QueryBuilder.swift

```diff
@@ -9,24 +9,24 @@
 import Foundation
 
 class Query {
-    let search: String?
-    let sort: QSort?
-    let invert: Bool?
-    let page: String?
-    let notebook: [QNotebook]?
-    let craft: [QCraft]?
-    let availability: [QAvailability]?
-    let weight: [QWeight]?
+    var search: String
+    var sort: QSort?
+    var invert: Bool?
+    var page: String?
+    var notebook: [QNotebook]
+    var craft: [QCraft]
+    var availability: [QAvailability]
+    var weight: [QWeight]
     
     init (
-        search: String? = "",
+        search: String = "",
         sort: QSort? = QSort.best,
```

Comment:
Looks like `sort` shouldn't be optional

Reply:
Removed for now, it's possible to search without specifying sort but more testing needed to see how results are ordered